pkg/pgsql/server/fmessages/string_reader.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2021 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fmessages
+
+import (
+	"bufio"
+	"encoding/binary"
+)
+
+func getNextString(r *bufio.Reader) (string, error) {
+	s, err := r.ReadBytes(0)
```

Comment:
is the intention to fully read the data as a string?

Reply:
yes, it's used to parse client messages